jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/BuildImageMojo.java

```diff
@@ -55,6 +55,20 @@
   /** {@code User-Agent} header suffix to send to the registry. */
   private static final String USER_AGENT_SUFFIX = "jib-maven-plugin";
 
+  /** Attempts to infer a known credential helper name from a specified registry. */
+  @VisibleForTesting
+  @Nullable
+  static String inferCredentialHelperName(String registry) {
+    if (registry.endsWith("gcr.io")) {
+      return "gcr";
+
+    } else if (registry.endsWith("amazonaws.com")) {
+      return "ecr-login";
+    }
+    // TODO: Add more common credential helpers.
```

Comment:
nit: this TODO seems unnecessary.

Reply:
This is mostly as a note to add more inferences such as Azure's -acr-* credential helpers. Is it standard practice to not have a note like this?

Reply:
Yeah, everyone uses them differently. I would think it's better to file a bug than leave this here, I tend to lose track of TODOs.

Reply:
Filed https://github.com/google/jib/issues/79